src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -472,9 +482,19 @@ public override void OnInitialize()
         {
             base.OnInitialize();
             AccessibilityRole = Role.Dialog;
+        }
+
+        private void OnAddedToWindow(object sender, EventArgs e)
+        {
             Show(); // calls RegisterDefaultLabel(); Hide() will call UnregisterDefaultLabel()
```

Comment:
Perhaps `IsVisible()` should be checked? (In case someone adds a hidden popup to window and `Show()`s it later.)

Reply:
Good point! I will check this property before calling Show() here.

Reply:
It seems that this scenario is unreachable in current circumstances. After adding object to window its Visibility is set to true.